br-webui/index.js

```diff
@@ -776,29 +776,34 @@ function updateCPUStats () {
 
 		// If command fail, return no status
 		if (throttled[0] != "throttled") {
-			cpu_stats.cpu_status = "No status"
+			cpu_stats.cpu_status = "No status!"
```

Comment:
"Semicolons are required after every statement", from: https://github.com/bluerobotics/internal/blob/master/software/guidelines/style.md#language-specific

Reply:
it was already missing. The guideline was written after this code. https://github.com/bluerobotics/companion/issues/172

Reply:
fixed